code/modules/reqs/supplypacks.dm

```diff
@@ -641,6 +641,11 @@ ARMOR
 	contains = list(/obj/item/clothing/mask/gas/swat)
 	cost = 5
 
+/datum/supply_packs/armor/imager_goggle
+	name = "Optical Imager Goggles"
+	contains = list(/obj/item/clothing/glasses/imager_goggles)
+	cost = 5
```

Comment:
```suggestion
	cost = 3
```
considering this does literally nothing but remove the light plane i don't see how it being 5 points is needed but whatever.
3 gives you 5 per plasma mining tick and I think that's fairer than 5 points